lib/datadog/tracing/contrib/grpc/patcher.rb

```diff
@@ -19,8 +19,6 @@ def target_version
           end
 
           def patch
-            require_relative '../../propagation/grpc'
-            require_relative 'datadog_interceptor'
```

Comment:
These were just unnecessary and actually the responsibility of `intercept_with_datadog.rb`.